CHANGES.md

```diff
@@ -2,6 +2,31 @@
      This file is generated. Do not write release notes here.
      Notes for unreleased changes go in ./UNRELEASED.md -->
 
+## 0.8.4
+
+### Feature Model Checker
+
+ * new sequential model checker that is using TransitionExecutor, see #467
+ * new command-line options, see #467 and the manual for details:
+   - choose the algorithm: `--algo=(offline|incremental)`
+   - pre-check, whether a transition disabled, discard the disabled transitions: `--discard-disabled`
+   - do not check for deadlocks: `--no-deadlock`
+   - pass tuning parameters in CLI: `--tune-here`
+
+### Feature TLA+ Parser (SANY importer)
```

Comment:
@konnov is this actually the format we want to use? I think a more common format would be

## Features

 * Model checker
   * Foo
   * Bar
   * Baz
 * TLA+ Parse
    * Bop
    * Bing
```

Or perhaps not even having a nested list. I"m fine with whatever you prefer, but want to confirm this what you actually intend.

Reply:
Yes, your format is better. I think I just once copied from the template :)

Reply:
Fixed the template. It was causing a  misunderstanding  apparently.